plugin/src/main/java/io/jenkins/plugins/coverage/model/CoverageTableModel.java

```diff
@@ -206,12 +215,12 @@ public int getLoc() {
         protected DetailedCell<?> createColoredCoverageColumn(final Coverage coverage, final String tooltip) {
             if (coverage.isSet()) {
                 double percentage = coverage.getCoveredPercentage().getDoubleValue();
-                DisplayColors colors = CoverageLevel.getDisplayColorsOfCoverageLevel(percentage, COLOR_PROVIDER);
+                DisplayColors colors = CoverageLevel.getDisplayColorsOfCoverageLevel(percentage, colorProvider);
                 String cell = div().withClasses(COVERAGE_COLUMN_OUTER).with(
                                 div().withClasses(COVERAGE_COLUMN_INNER)
                                         .withStyle(String.format(
                                                 "background-image: linear-gradient(90deg, %s %f%%, transparent %f%%);",
-                                                colors.getFillColorAsHex(),
+                                                colors.getFillColorAsRGBAHex(80),
```

Comment:
80 seems to be some kind of random, maybe it helps to define a constant with this value that uses a appropriate name?